### ISSUE TITLE
NTR: Alkyl-DHAP synthase low in fibroblast (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9288]
New term request Alkyl-DHAP synthase low in fibroblast (segal_070726205427): Alkyldihydroxyacetonephosphate synthase  UMLS: 4479029

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000427
- label: Reduced alkyl-dihydroxyacetonephosphate synthase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:9553082
- def: Activity of alkyl-dihydroxyacetonephosphate (DHAP) synthase below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9288
- synonyms: Reduced alkyl-DHAP synthase activity in cultured fibroblasts(NONE)
